assets/js/components/Work/Tabs/Preservation.jsx

```diff
@@ -21,20 +19,32 @@ const WorkTabsPreservation = ({ work }) => {
             </tr>
           </thead>
           <tbody>
-            <tr>
-              <td>AM</td>
-              <td>1234_12.tif</td>
-              <td>ASDF09860986AS0986ASDF</td>
-              <td>s3://asoyuoasd.com</td>
-              <td>
-                <FontAwesomeIcon icon="check" />
-              </td>
-              <td>
-                <button className="button">
-                  <FontAwesomeIcon icon="trash" />
-                </button>
-              </td>
-            </tr>
+
+            { work.fileSets && work.fileSets.length > 0 &&
```

Comment:
No big deal, but just for the sake of practice, let's make one change to the PR code...   Here, the condition `work.fileSets.length > 0` technically isn't necessary since if the length === 0, any iterators on the array will just run 0 times.  How about you remove that and re-push?